### PR TITLE
Create how-to-create-multiselect-checkbox-parameter-editor.md

### PR DESCRIPTION
**Pull request by DimitarNikolovv06**

### CODE REVIEW

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+
+## Description
+The article elaborates how to change the default editors for visible multivalue parameters in the HTML5 Viewer's Parameters Area.
+
+Custom parameter editors are defined through the **parameterEditors** [(Report Viewer Initialization)](https://docs.telerik.com/reporting/html5-report-viewer-jquery-fn-telerik-reportviewer) array passed as an option when creating the report viewer widget. Each object represents a parameter editor factory for creating editors suitable to edit a specific report parameter configuration.
+
+Each editor is an object which contains two methods: **match** and **createEditor**.
+
+The **match** method accepts a report parameter to be edited as an argument and returns a boolean value which indicates whether the parameter editor is suitable for this parameter. The parameter variable exposes the properties of the report parameter like name, allowNull, availableValues, multiValue, type and etc.
```
Comment (todorarabadzhiev) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:31]: consider removing 'and' before 'etc.'

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+The main work for creating and utilizing the parameter editor is done in the **createEditor** method. Its purpose is to create the parameter editor UI and wire it to the **parameterChanged** callback when a new value is selected. The return result is a new object containing the **beginEdit** method which is ttry point for creating the editor from the viewer.
```
Comment (todorarabadzhiev) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:33]: correct 'ttry point'

**Review by todorarabadzhiev — APPROVED** [review]

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+title: How to Create MultiSelect CheckBox Parameter Editor
+description: Use kendo's CheckBoxGroup widget with multivalue parameters
```
Comment (nelci592) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:3]: You can consider changing it to The article explains how to display a multivalue parameter by using the CheckBoxGroup widget from Kendo UI

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+		<tr>
+			<td>Report Viewer</td>
+			<td>HTML5</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article elaborates how to change the default editors for visible multivalue parameters in the HTML5 Viewer's Parameters Area.
```
Comment (nelci592) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:25]: **on** how

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+Custom parameter editors are defined through the **parameterEditors** [(Report Viewer Initialization)](https://docs.telerik.com/reporting/html5-report-viewer-jquery-fn-telerik-reportviewer) array passed as an option when creating the report viewer widget. Each object represents a parameter editor factory for creating editors suitable to edit a specific report parameter configuration.
```
Comment (nelci592) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:27]: You can use ../reporting/html5-report-viewer-jquery-fn-telerik-reportviewer

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+The **match** method accepts a report parameter to be edited as an argument and returns a boolean value which indicates whether the parameter editor is suitable for this parameter. The parameter variable exposes the properties of the report parameter like name, allowNull, availableValues, multiValue, type, etc.
+
+The main work for creating and utilizing the parameter editor is done in the **createEditor** method. Its purpose is to create the parameter editor UI and wire it to the **parameterChanged** callback when a new value is selected. The return result is a new object containing the **beginEdit** method which is the entry point for creating the editor from the viewer.
+
+The following example illustrates how to use the [Kendo CheckBoxGroup](https://docs.telerik.com/kendo-ui/controls/editors/checkboxgroup/overview) widget for a multi parameter value parameter editor which also has available values.
+
+## Solution
+
+1. Add a reference to kendo of version R1 2021 at least
```
Comment (nelci592) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:39]: Use capital letter for Kendo

File: knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+                    sendEmail: { enabled: true },
+                });
+
+        });
+```
+
+## See Also
+[What's new in Kendo UI for jQuery](https://www.telerik.com/blogs/whats-new-kendo-ui-jquery-r1-2021) 
+
+[How to Create a Custom Parameter Editor](https://docs.telerik.com/reporting/html5-report-viewer-howto-custom-parameter-editor)
```
Comment (nelci592) [inline knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md:107]: Again, you can use relative path ../reporting/html5-report-viewer-howto-custom-parameter-editor

**Review by madsfarner — APPROVED** [review]

**Pull request merged**